pic-sure-hpds-ui/src/main/webapp/picsureui/header/header.js

```diff
@@ -109,35 +110,68 @@ define(["jquery", "backbone","handlebars", "text!header/header.hbs", "overrides/
             sel.addRange(range);
             document.execCommand("copy");
 
-            $("#user-token-copy-button").html("COPIED");
+            $("#user-token-copy-button").html("Copied");
 
             document.getElementById("user_token_textarea").textContent
                 = document.getElementById("user_token_textarea").value
                 = originValue;
         },
-        refreshToken: function(){
-            notification.showConfirmationDialog(function () {
+        refreshToken: function(event){
+		    $('#user-token-refresh-button').hide();
+		    $('#user-token-refresh-confirm-container').show();
+            $('#user-token-yes-button').focus();
+
+            $('#user-token-no-button').click(function(e) {
+                $('#user-token-refresh-confirm-container').hide();
+                $('#user-token-refresh-button').show();
+                $('#user-token-refresh-button').focus();
+                e.preventDefault();
```

Comment:
Added these focus and prevent default to prevent focus from going to the clicked button which becomes hidden